src/Microsoft.AspNetCore.Mvc.RazorPages/baseline.netcore.json

```diff
@@ -1,404 +1,437 @@
 {
-  "AssemblyIdentity": "Microsoft.AspNetCore.Mvc.RazorPages, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
+  "AssemblyIdentity": "Microsoft.AspNetCore.Mvc.RazorPages, Version=2.0.3.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
   "Types": [
     {
-      "Name": "Microsoft.AspNetCore.Mvc.Internal.MvcRazorPagesDiagnosticSourceExtensions",
+      "Name": "Microsoft.AspNetCore.Mvc.Filters.IAsyncPageFilter",
```

Comment:
Removing the pub-internal bits seems to be reordering / confusing the baseline here too.